### PR TITLE
Fix Loading Customer Data

### PR DESCRIPTION
**Pull request by PaitoAnderson**
Test Procedure:
1) Login
2) Reload Page
3) Check if can still can load `/account`

### CODE REVIEW

**Comment by jaepass** [comment]
Nice thanks for the PR! Works well with persisting the customer! There's a slight delay in checking if the customer exists when refreshing the page so there's a split second where the logged out screen shows. It's pretty minor but I'm not sure what the fix is at this point.

**Review by jaepass — CHANGES_REQUESTED** [review]
Nice, thanks for the PR! Works well with persisting the customer! There's a slight delay in checking if the customer exists when refreshing the page so there's a split second where the logged out screen shows. It's pretty minor but I'm not sure what the fix is at this point.

**Comment by PaitoAnderson** [comment]
@jaepass Good point, I didn't notice that.

Let me know if you think this is a good approach, my thought process was these parts are far apart so I definitely need to use redux, but I didn't want to mix a `isLoading` prop that's only for UI in the `customer` data, so I created another `loading` object that can be potentially be used for keeping the loading state of other redux items too.

**Comment by jaepass** [comment]
Awesome, thank you @PaitoAnderson!! Let me test this out as soon as I can!

File: pages/account/index.js [hunk 1]
```diff
@@ -248,6 +248,11 @@ class CustomerAccountPage extends Component {
   }
 
   render() {
+
+    if (this.props.loading.customer) {
+      return <TemplatePage page={ {message: 'Loading'} } />
```
Comment (jaepass) [inline pages/account/index.js:253]:
```suggestion
      return <TemplatePage page={  { message: 'Loading...' }  } />
```

File: pages/account/[id].js [hunk 2]
```diff
@@ -124,21 +133,11 @@ export default function SingleOrderPage() {
...
-        </Head>
-        <div className="py-5 my-5 text-center">
-          <h4 className="mt-4">{ data.message }</h4>
-        </div>
-        <Footer />
-      </Root>
-    )
-  };
+  if (customerLoading) {
+    return <TemplatePage page={ {message: 'Loading'} } />
```
Comment (jaepass) [inline pages/account/[id].js:139]:
```suggestion
    return <TemplatePage page={  { message: 'Loading...' }  } />
```

**Review by jaepass — COMMENTED** [review]
LGTM, I've just added some minor changes. Happy to merge after! Thank you again for the contributions!

**Comment by PaitoAnderson** [comment]
Thanks for the quick review!

**Pull request merged**